application/frontend/config/main.php

```diff
@@ -91,6 +91,7 @@
             ]
         ],
         'urlManager' => [
+            'cache' => null,
```

Comment:
"Set this property to false or null if you do not want to cache the URL rules."
https://www.yiiframework.com/doc/api/2.0/yii-web-urlmanager

Reply:
What does not-caching URL rules gain us?

Reply:
I believe this is specifying the default, which we have been using implicitly. Specifying it here silences a warning message.